addons/io_hubs_addon/preferences.py

```diff
@@ -88,52 +75,14 @@ class UninstallDepsOperator(bpy.types.Operator):
     bl_property = "dep_names"
```

Comment:
I'm not actually sure whether this bl_property specification does anything here, but it shows an error in a terminal with the changes.

Reply:
I don't remember what is that doing there but I tested and removing it doesn't seem to affect so I've removed it.